docker/Dockerfile-pytorch

```diff
@@ -0,0 +1,22 @@
+ARG base_image_tag
+FROM twosixarmory/base:${base_image_tag} as armory-base
+ARG armory_version
+ARG armory_build_type
+
+FROM armory-base AS armory-local
+# Putting here to see if it will cache
+# TODO this is still not cache
+RUN /opt/conda/bin/pip install --no-cache-dir \
+    adversarial-robustness-toolbox==1.9.1
```

Comment:
The other containers should follow this pattern of installing ART as the first step in `armory-local` layer. That will enable easier incorporation of ART patch updates, which historically have been necessary.